addon/components/stacked-vertical-bar-chart.js

```diff
@@ -0,0 +1,1015 @@
+      // If 0 or 1 slice labels were filtered out, we can just show all slice
+      // labels explicitly. We only want the 'Other' slice if it has at least
+      // 2 slice labels contained aggregated inside.
+      return _.pluck(largestSliceData, 'sliceLabel');
+    } else {
+      // Otherwise, just return the slice labels that passed the filters.
+      return _.pluck(nonOtherSlices, 'sliceLabel');
+    }
+  }),
+
+  /**
+   * The sliceLabels that will be aggregated into the 'Other' slice and not
+   * explicitly shown in the legend.
+   * @type {Array.<string>}
+   */
+  otherSliceTypes: Ember.computed('largestSliceData.[]', 'nonOtherSliceTypes.[]', function() {
+    var allSliceTypes = _.pluck(this.get('largestSliceData'), 'sliceLabel');
+    return _.difference(allSliceTypes, this.get('nonOtherSliceTypes'));
+  }),
+
+  /**
+   * Input data mapped by barLabel AFTER 'Other' slices have been calculated
+   * and with slices sorted correctly for each bar. Bar sorting is handled by
+   * `barNames`, but slice sorting is handled here.
+   * Key: barLabel
+   * Value: Array of [sorted] slice objects (sliceLabel, barLabel, value)
+   * @type {Object.<string, Array.Object>>}
+   */
+  sortedData: Ember.computed('dataGroupedByBar', 'otherSliceLabel', 'nonOtherSliceTypes.[]', 'sliceSortingFn', function() {
+    var groupedData, nonOtherSliceTypes, otherSliceLabel;
+    groupedData = this.get('dataGroupedByBar');
+    nonOtherSliceTypes = this.get('nonOtherSliceTypes');
+    otherSliceLabel = this.get('otherSliceLabel');
+    return _.reduce(groupedData, (result, barData, barLabel) => {
+      // Create an empty 'Other' slice. Go through every slice in each bar
+      // and look for slices that need to be aggregated into 'Other', updating
+      // the value of the otherSlice along the way.
+      var newBarData, otherSlice;
+      newBarData = [];
+      otherSlice = { barLabel: barLabel,
+                     sliceLabel: otherSliceLabel,
+                     value: 0 };
+      barData.forEach((slice) => {
+        if (nonOtherSliceTypes.indexOf(slice.sliceLabel) !== -1) {
+          newBarData.push(slice);
+        } else {
+          otherSlice.value += slice.value;
+        }
+      });
+      newBarData.sort(this.get('sliceSortingFn'));
+      if (otherSlice.value !== 0) {
+        newBarData.push(otherSlice);
+      }
+      result[barLabel] = newBarData;
+      return result;
+    }, {});
+  }),
+
+  /**
+   * Final data to be consumed by d3 and rendered into the chart.
+   * Contains positioning information of slice above/below x-axis, labels,
+   * and color.
+   * @type {Array.<Object>}
+   */
+  finishedData: Ember.computed('sortedData', 'otherSliceLabel', function() {
+    var posTop, negBottom, stackedValues, otherSliceLabel;
+    otherSliceLabel = this.get('otherSliceLabel');
+    return _.map(this.get('sortedData'), (values, barLabel) => {
+      posTop = 0;
+      negBottom = 0;
+      stackedValues = _.map(values, function(d) {
```

Comment:
Agreed, this was another artifact from VBC. I will update the variable names here